app/core/utils.coffee

```diff
@@ -457,8 +457,9 @@ startsWithVowel = (s) -> s[0] in 'aeiouAEIOU'
 filterMarkdownCodeLanguages = (text, language) ->
   return '' unless text
   currentLanguage = language or me.get('aceConfig')?.language or 'python'
-  excludedLanguages = _.without ['javascript', 'python', 'coffeescript', 'clojure', 'lua', 'java', 'io', 'html'], currentLanguage
-  # Exclude language-specific code blocks like ```python (... code ...)``` for each non-target language.
+  excludedLanguages = _.without ['javascript', 'python', 'coffeescript', 'lua', 'java', 'cpp', 'html'], currentLanguage
```

Comment:
Is there a possibility there is a `clojure` or `io` language that isn't excluded after this change?

Reply:
I'll do a check and delete any if so.